psi4/src/psi4/libpsipcm/psipcm.cc

```diff
@@ -57,7 +57,7 @@ std::pair<std::vector<double>, std::vector<double>> collect_atoms(std::shared_pt
     int nat = molecule->natom();
     std::vector<double> charges(nat);
     for (int i = 0; i < nat; ++i) {
-        charges[i] = molecule->Z(i);
+        charges[i] = molecule->ftrue_atomic_number(i);
```

Comment:
You plan to change this from `ftrue_atomic_number` -> `true_atomic_number`? That sounds good -- `full_atoms` at best refers back to the original pre-nbody molecule. And `natom`, `true_atomic_number`, `geometry` should be all the same with or without `f` prefix.

Reply:
change to `true_atomic_number`